src/screens/ReferFriends/ReferralIncomingReward.js

```diff
@@ -85,7 +85,7 @@ class ReferralIncomingReward extends React.PureComponent<Props> {
     const relatedCampaign = rewardsByCampaign['1world'] ? '1world' : 'pillar';
 
     const rewardText = getRewardText(rewardsByCampaign, relatedCampaign);
-    const isGettingReward = rewardText;
+    const isGettingReward = !!rewardText;
```

Comment:
{!!isGettingReward && <RewardBadge source={rewardBadge} />}
this is already cast at line 101

Reply:
I'll remove '!!' in later lines. Keeping it as boolean here :)